app/views/comments/_hide_comments_modal.html.erb

```diff
@@ -0,0 +1,19 @@
+<div id="hide-comments-modal" class="hidden">
+  <%= form_with url: "/comments/hide", class: "hide-comments-modal__form", id: "hide-comments-modal__form", method: :patch, remote: true, html: { "data-type": "json" } do |f| %>
+    <div class="hide-comments-modal__content">
+      <p class="pb-2">
+        Are you sure you want to hide this comment? It will become hidden in your post, but will still be visible via the comment's permalink.
+      </p>
+      <label class="crayons-field crayons-field--checkbox my-2">
+        <%= f.check_box :hide_children, class: "hide_children crayons-checkbox" %>
+        <p class="crayons-field__label">Hide child comments as well</p>
+      </label>
+      <p class="pb-4 pt-2">
+        <button type="submit" class="crayons-btn" aria-label="<%= t("views.comments.hide.confirm") %>">
+          Confirm
+        </button>
+      </p>
+    </div>
+  <% end %>
+  <p class="fs-s color-base-60">For further actions, you may consider blocking this person and/or reporting abuse</p>
```

Comment:
(suggestion / not blocking) - I think it would be nice to provide links to how to block/report abuse here

Reply:
I will keep this improvement for the next pr.